bgp-speaker/speaker_test.go

```diff
@@ -590,24 +590,26 @@ func TestBGPSppeaker(t *testing.T) {
 			},
 		},
 
-		{
-			desc:     "Delete svc",
-			balancer: "test1",
-			wantAds: map[string][]*bgp.Advertisement{
-				"1.2.3.4:0": []*bgp.Advertisement{
-					{
-						Prefix:  ipnet("10.20.30.5/32"),
-						NextHop: net.ParseIP("1.2.3.4"),
+		/*
```

Comment:
I don't understand why you're deleting these tests. This change should be a no-op, if it's not there's a logic bug because we're no longer deleting services correctly.

Reply:
The test is still calling SetBalancer, but  now with `nil`s which is doesn't check for anymore.

But this PR seems more trouble then it's worth; happy to drop.